tests/RobotFramework/tests/cumulocity/configuration/backward_compatibility.robot

```diff
@@ -0,0 +1,25 @@
+*** Settings ***
+Resource            ../../../resources/common.resource
+Library             ThinEdgeIO
+Library             Cumulocity
+
+Test Teardown      Get Logs    name=${DEVICE_SN}
+
+Test Tags          theme:configuration    theme:installation
+
+*** Test Cases ***
+
+Migrate Legacy Configuration Files
+    ${DEVICE_SN}=    Setup    skip_bootstrap=${True}
```

Comment:
Any reason why you use `${True}` instead of `True` for `skip_bootstrap`? From what I read, `${True}` passes the value as a string where as the `skip_bootstrap` arg is annotated as `boolean` type. Somehow both seems to work. Just wanted to know if this is better for some reason.